helm/values.yaml

```diff
@@ -508,7 +508,7 @@ server:
       type: pd-standard
 
   logging:
-    level: error
+    level: warn
```

Comment:
it already is set to `error` in helm-values repo.

Reply:
We will adjust in helm-values repo as needed

Reply:
https://github.com/Traceableai/helm-values/pull/4139

Reply:
what I meant was that we already have set the logging level to `error`. are there any specific warn logs we are interested in? there were too much logging by pinot.

Reply:
Laxman and mihir were of opinion we should have warn logs and fix any underlying issues

Reply:
I only suggested better to keep warn as the default.
If they are too noisy, we may need to investigate and fix or turn them off at package level in worst case.